liquibase-core/src/main/java/liquibase/changelog/filter/ExecutedAfterChangeSetFilter.java

```diff
@@ -26,9 +26,9 @@ public ExecutedAfterChangeSetFilter(Date date, List<RanChangeSet> ranChangeSets)
     @Override
     public ChangeSetFilterResult accepts(ChangeSet changeSet) {
         if (changeLogsAfterDate.contains(changeSet.toString())) {
-            return new ChangeSetFilterResult(true, "Changeset ran after "+ new ISODateFormat().format(new java.sql.Timestamp(date.getTime())), this.getClass());
+            return new ChangeSetFilterResult(true, "Changeset ran after "+ new ISODateFormat().format(new java.sql.Timestamp(date.getTime())), this.getClass(), getMdcName());
         } else {
-            return new ChangeSetFilterResult(false, "Changeset ran before "+ new ISODateFormat().format(new java.sql.Timestamp(date.getTime())), this.getClass());
+            return new ChangeSetFilterResult(false, "Changeset ran before "+ new ISODateFormat().format(new java.sql.Timestamp(date.getTime())), this.getClass(), getMdcName());
         }
     }
 }
```

Comment:
In the summary output, we are referring to the DBMS filter as "mismatched DBMS".  We might consider doing that here.